tests/tests_tui/test_tui_directorytree.py

```diff
@@ -245,3 +245,68 @@ def set_signal_to_path(path_):
             )
 
             await pilot.pause()
+
+    @pytest.mark.asyncio
+    async def test_create_folders_directorytree_rename(
+        self, setup_project_paths
+    ):
+        tmp_config_path, tmp_path, project_name = setup_project_paths.values()
+
+        rawdata_path = tmp_path / "local" / project_name / "rawdata"
+
+        app = TuiApp()
+        async with app.run_test(size=self.tui_size()) as pilot:
```

Comment:
Nice! This test looks good, one additional suggestion is to check the file is actually changed on disk correctly also. For example, that after renaming there is now no folder called `sub-001`, but there is called `sub-002` for example. Otherwise this is good to go!